deploy/crds/keycloak.org_keycloakbackups_crd.yaml

```diff
@@ -60,6 +60,14 @@ spec:
                     a CronJob
                   type: string
               type: object
+            imageOverrides:
```

Comment:
I believe it's not necessary here - we can just rely on whatever image is specified in `Keycloak` CR.

Reply:
Thank you for your comment, @slaskawi 
Sorry, I am not clear here. 
Function `postgresqlAwsBackupCommonContainers` is using the BackupImage as follows:
`func postgresqlAwsBackupCommonContainers(cr *v1alpha1.KeycloakBackup) []v1.Container {
	return []v1.Container{
		{
			Name:    cr.Name,
			Image:   BackupImage
                         ...`
Here we can find that this function only provides one parameter `cr *v1alpha1.KeycloakBackup`.  If we specify BackupOverrideImage in `cr *v1apha1.Keycloak`, how can this function receive the overrided Backup Image?

Reply:
I just realized, that `KeycloakBackup` CR doesn't have `instanceSelector` field as `KeycloakRealm` CR. I believe it was just overlooked and should be implemented. 

So my suggestion would be to add a similar mechanism (`instanceSelector`, see [here](https://github.com/keycloak/keycloak-operator/blob/437d680c764abcb6df02b9278f3d50f4f7ed57a1/deploy/examples/realm/basic_realm.yaml#L13-L15) for example)  for finding a corresponding `Keycloak` CR from `KeycloakBackup` CR (similarly how `Keycloak` CR is referenced from `KeyclaokRealm` CR). 

Once that's done - you should be able to modify the code in such a way, so that `postgresqlAwsBackupCommonContainers` receives both `KeycloakBackup` and `Keycloak` CRs.

Sorry, that this task grew but I hope it's not a very complicated task to be implemented (most of the bits could be copy-pasted, or better - extracted, from `keycloakrealm_controller.go`).